front-end/src/components/Manager/Main/EntitiesMenu/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Something is not right. I see a few files copied from Admin component to Manager component. They do not belong here. Please remove them if you added them by mistake.

Reply:
There are only necessary fixes. I didn't copy any files from Admin to Manager.
In manager component only replace console.log with toastr.
If i missed something please show which files.

Reply:
Ah, I see. You moved them into 'Main' folder. For some reason they appear here as new files.